app/(backend)/factories/serviceFactory.ts

```
@@ -0,0 +1,83 @@
+    );
+  }
+  
+  /**
+   * Reset all service instances (useful for testing)
+   */
+  static resetServices(): void {
+    this.providerService = null;
+    this.apiKeyService = null;
+    this.modelService = null;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)